src/main/java/net/minecraftforge/gradle/tasks/abstractutil/ExtractTask.java

```diff
@@ -173,14 +170,24 @@ public void exclude(Closure<Boolean> c) {
     }
 
     public FileCollection getSourcePaths() {
-        FileCollection collection = getProject().files(new Object[]{});
+        FileCollection collection = getFiles();
 
         for (DelayedFile file : sourcePaths)
-            collection = collection.plus(getProject().files(file));
+            collection = collection.plus(getFiles(file));
 
         return collection;
     }
 
+    private FileCollection getFiles(Object... paths) {
```

Comment:
getFiles -> createFiles/createFileCollection is better I think.

Reply:
now createFileCollection